src/status_im/ui/screens/browser/events.cljs

```diff
@@ -58,6 +61,20 @@
      (-> (add-browser-fx cofx new-browser)
          (update-in [:db :browser/options] #(assoc % :browser-id (:browser-id new-browser)))))))
 
+(handlers/register-handler-fx
+ :update-browser-with-history
+ [re-frame/trim-v]
+ (fn [{:keys [now] :as cofx} [browser url]]
+   (let [history-index (if (nil? (get browser :history-index)) 0 (get browser :history-index))
+         history (get browser :history)
```

Comment:
let's just use `(:history browser)` https://github.com/bbatsov/clojure-style-guide#keywords-as-fn-to-get-map-values
or destruct `browser` in params, even better imo

Reply:
Done!